qos/cometbft/endpoint.go

```diff
@@ -11,8 +11,11 @@ import (
 var (
 	errNoHealthObs           = fmt.Errorf("endpoint has not had an observation of its response to a health check request")
```

Comment:
Wdyt of renaming and grouping these more appropriately (for future maintenance and additions)?

```
errHealthReqNoObs
errHealthReqInvalidObs
errHealth...

errStatusInvalidChainId
errStatusInvalidObs
errStatus...
```